### DISCUSSION TITLE
How can i disable --cache?

### DISCUSSION THREAD

**Discussion by uebmaster** [EasyEngine/easyengine#1730] [Support Requests]
I've installed wp with `--proxy-cache=on`

But i've disabled cause i can't see the change in the public view.

After running:

`sudo ee site update domain.com --proxy-cache=off`

the proxy cache is off but the cache is still enable:

![image](https://user-images.githubusercontent.com/5565506/188744947-c0ece684-9337-478c-898c-7240a9e83bb9.png)

How can i disable cache?

Thanks in advance.

**Comment by marcoandrei** [comment]
Have you tried to restart the site? (https://easyengine.io/commands/site/restart/)
